### ISSUE TITLE
Password reset not working

### ISSUE THREAD

**Issue by kruisdraad** [AbuseIO/AbuseIO#299]
The password reset is not fully working. You can follow the reset and be logged in, however the newly created password is not working on the reset password screen. When changing the password thru the profile works.

Issue is sourced with our user model, as we implemented our own auth we make user thru a model's mutator that a password is properly encrypted. However the reset blade and methods ALSO encrypts when resetting the password. This double crypt will void any password entered on the reset method.

Walkarround:
If you face this issue, please note that you are logged in if you use a valid reset token and will keep that session. After changing the password visit /admin/home and change the password thru the profile again to make the new password valid.

You can also fix this by patching the laravel code:

```
./vendor/laravel/framework/src/Illuminate/Foundation/Auth/ResetsPasswords.php

    protected function resetPassword($user, $password)
    {
        //$user->password = bcrypt($password);
        $user->password = $password;

        $user->save();

        Auth::login($user);
    }



./vendor/laravel/framework/src/Illuminate/Foundation/Auth/RedirectsUsers.php

-        return property_exists($this, 'redirectTo') ? $this->redirectTo : '/home';
+        return property_exists($this, 'redirectTo') ? $this->redirectTo : '/admin/home';

```

This also fixes the broken redirect after changing the password. Please note that these changes in the vendor dir are overwritten on any update with composer.

**Comment by kruisdraad** [comment]
obviously we will fix this in the next version ;)

**Comment by kruisdraad** [comment]
implemented in 4.3.0
